aep/general/0155/aep.md

```diff
@@ -0,0 +1,106 @@
+# Idempotency
+
+It is sometimes useful for an API to have a unique, customer-provided
+identifier for particular requests. This can be useful for several purposes,
```

Comment:
nit: switch to a list form, to make it easy to see the reasons?

Reply:
Done.